src/js/components/dashboard/DashboardTable.jsx

```diff
@@ -229,13 +246,18 @@ export default class DashboardTable extends React.Component {
             loadingClass = ' loading';
         }
 
-        let newHeaders = tableHeaders;
+        let newHeaders = tableHeadersActive;
+
+        //cannot be added to the const because if a user is read only then delete will not be created
 
-        if(PermissionsHelper.checkAgencyPermissions(this.props.session)){
-            newHeaders = tableHeaders.concat('Delete');
+        if(PermissionsHelper.checkAgencyPermissions(this.props.session) && !this.props.isCertified){
+            newHeaders = tableHeadersActive.concat('Delete');
+        } else if(this.props.isCertified) {
+            newHeaders = tableHeadersCertified;
         }
-        if(this.props.isCertified) {
-            newHeaders = tableHeaders.concat(["Certified By","Certified On"]);
+        let unsortable = [0,5,6]
+        if(this.isCertified){
```

Comment:
Did you test sorting by certified date/certifying user? If those don't work we might want to make it unsortable by them for now